src/components/nav.css

```diff
@@ -3,63 +3,54 @@
 nav {
   padding: 0 var(--base-content-padding);
   z-index: 200;
-}
-nav a {
-  padding: 12px 0 8px;
-  border-bottom: 3px solid transparent;
-  display: block;
-  font-size: 16px;
-  text-decoration: none;
-  font-weight: 600;   
-  text-decoration: none;
-  text-transform: uppercase;
-  color: #777;
+  background-color: #fff;
+  border-top: 1px solid #efefef;
 }
 .navbar {
   display: flex;
-  justify-content: flex-start;
   flex: 1 0 100%;
+  flex-direction: row;
+  justify-content: flex-start;
   list-style: none;
 }
-
-.navbar-item {
+nav ul li {
   position: relative;
   margin: 0;
   padding: 0 2rem 0 0;
 }
-
-/* header-specific nav styles */
-header nav {
-  border-top: 1px solid #efefef;
-  background-color: #fff;
-}
-header nav a {
-    color: #777;
-    font-weight: 600;
-    text-decoration: none;
-    text-transform: uppercase;
-}
-header nav a:hover, header nav a.active {
-  border-bottom-color: rgb(183,191,16);
+nav a {
+  padding: 12px 0 8px;
+  border-bottom: 3px solid transparent;
+  display: block;
+  font-size: 18px;
+  font-weight: 600;   
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #555;
+  letter-spacing: 1px;
```

Comment:
Minor "code smell" to choose a different typeface but I don't mind this small tweak for readability.

Reply:
Agreed. Perhaps we can address this when we revisit how we want to handle fonts/typefaces.